app/src/org/commcare/activities/UpdateActivity.java

```diff
@@ -205,7 +205,8 @@ protected void startUpdateCheck() {
         }
 
         String ref = ResourceInstallUtils.getDefaultProfileRef();
-        updateTask.execute(ref);
+        String newestRef = ResourceInstallUtils.getLatestProfileRef();
```

Comment:
I'm guessing we would want this in an alternative update path, as opposed to replacing the default update behavior